classes/RoutesManager.php

```diff
@@ -12,9 +12,11 @@ class RoutesManager
     private const ROUTE_PATH = __DIR__ . '/../routes.php';
     private const ROUTE_TEMPLATE_PATH = __DIR__ . '/../templates/route.tpl';
     private const ROUTE_TEMPLATE_PLACEHOLDERS = [
+        '%router_method%',
         '%base_endpoint%',
         '%controller_class%',
         '%options%',
+        '%middleware_string%',
```

Comment:
Referencing [this comment](https://github.com/iPriceGroup/ocapiplugin/pull/21#discussion_r412153016) - let's call this `route_methods` as that's what the placeholder is generally for. Not only middlewares.